core/src/main/java/org/opensearch/sql/ast/dsl/AstDSL.java

```diff
@@ -249,27 +247,19 @@ public static Function function(String funcName, UnresolvedExpression... funcArg
   }
 
   /**
-   * CASE
-   *     WHEN search_condition THEN result_expr
-   *     [WHEN search_condition THEN result_expr] ...
-   *     [ELSE result_expr]
-   * END
+   * CASE WHEN search_condition THEN result_expr [WHEN search_condition THEN result_expr] ... [ELSE
```

Comment:
another javadoc to fix

Reply:
https://github.com/diffplug/spotless/issues/1501 Currently possible with `--skip-javadoc-formatting` using Google Java Formatter, but not _yet_ through Spotless. I propose we move over to a custom XML config.... sample PRs will follow. Briefly looking at changes, none seem to impact this sort of JavaDoc!

Reply:
Good. We can groom javadocs manually later if needed.

Reply:
https://github.com/opensearch-project/sql/issues/1913 Added issue for it.